region/lowernorfair/east.json

```diff
@@ -970,14 +1006,75 @@
           "to": [
             {
               "id": 2,
-              "requires": [ "canPassBombPassages" ],
+              "requires": [
+                {"or": [
+                  {"and": [
+                    "heatProof",
+                    {"or": [
+                      "canUseMorphBombs",
+                      {"and": [
+                        "canUsePowerBombs",
+                        {"cost": [
+                          {"ammo": [{
+                            "type": "PowerBomb",
+                            "count": 2
+                          }]}
+                        ]}
+                      ]}
+                    ]}
+                  ]},
+                  {"and": [
+                    "canHeatRun",
+                    "canUsePowerBombs",
+                    {"cost": [
+                      {"heatFrames": 500},
+                      {"ammo": [{
+                        "type": "PowerBomb",
+                        "count": 2
+                      }]}
+                    ]}
+                  ]}
+                ]}
+              ],
               "unlock": null
             },
             {
               "id": 5,
-              "requires": [ "canKillRedKiHunters" ],
+              "requires": [
+                {"or": [
+                  {"and": [
+                    "heatProof",
+                    "canKillRedKiHunters"
```

Comment:
If taking KiHunter hits instead of killing them is expected for a heat run, surely it should also be a valid strat when heatRuns are disallowed but your are heatproof?

Reply:
Isn't that using Morph Ball though?

Reply:
I didn't use Morph Ball when I messed around with evading them. There is a reliable way to avoid the first KiHunter coming out of the save door (Angle up to stop momentum, then jump to bonk on the ceiling. Then run left, he *will* dive down, and you can run and jump above him to the right). The second guy is fairly easy to evade. The third guy usually requires taking a hit, and Morph actually hasn't helped me avoid him much.

Reply:
The first one is easy but there's a couple crappy RNG versions for the other two. One is where both the 2nd and 3rd is bunched up in the same place and blocks your path. Often yields 2 hits. Another one is where the 3rd is almost down at the middle section. 1 hit is the usual but I feel 2 happen too often.

Reply:
I have no issues with 2 hits.